[libcap] Build error on arm-linux

Package: libcap:arm-linux -> 2.69#2

**Host Environment**

- Host: arm-linux
- Compiler: GNU 10.2.1
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 3f3d8b003 2023-12-12 (22 hours ago)

**To Reproduce**

`vcpkg install `
**Failure logs**

```
-- Using cached libpcap-2.69.tar.gz.
-- Cleaning sources at /home/igor/drawme/src/vcpkg/buildtrees/libcap/src/libpcap-2-fe0a022327.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/igor/drawme/src/vcpkg/downloads/libpcap-2.69.tar.gz
-- Using source at /home/igor/drawme/src/vcpkg/buildtrees/libcap/src/libpcap-2-fe0a022327.clean
-- Getting CMake variables for arm-linux-dbg
-- Getting CMake variables for arm-linux-rel
-- Configuring arm-linux-dbg
-- Configuring arm-linux-rel
-- Building arm-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/make V=1 -j 5 -f Makefile.vcpkg all
    Working Directory: /home/igor/drawme/src/vcpkg/buildtrees/libcap/arm-linux-dbg/
    See logs for more information:
      /home/igor/drawme/src/vcpkg/buildtrees/libcap/build-arm-linux-dbg-out.log
      /home/igor/drawme/src/vcpkg/buildtrees/libcap/build-arm-linux-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/libcap/portfile.cmake:25 (vcpkg_install_make)
  scripts/ports.cmake:170 (include)



```
<details><summary>/home/igor/drawme/src/vcpkg/buildtrees/libcap/build-arm-linux-dbg-err.log</summary>

```
/bin/sh: 1: ./_makenames: not found
make[1]: *** [Makefile:82: cap_names.h] Error 127
make: *** [Makefile.vcpkg:16: libcap/cap_names.h] Error 2
```
</details>

<details><summary>/home/igor/drawme/src/vcpkg/buildtrees/libcap/build-arm-linux-dbg-out.log</summary>

```
/usr/bin/make -C libcap cap_names.h "AR=/usr/bin/arm-linux-gnueabihf-ar"     "BUILD_CC=touch a.out | touch conftest | true"     "CC=/usr/bin/arm-linux-gnueabihf-gcc"     "RANLIB=/usr/bin/arm-linux-gnueabihf-ranlib"     "lib=lib"     "prefix=/home/igor/drawme/src/vcpkg_installed/arm-linux/debug" SHARED=no
make[1]: Entering directory '/home/igor/drawme/src/vcpkg/buildtrees/libcap/arm-linux-dbg/libcap'
=> making cap_names.list.h from /home/igor/drawme/src/vcpkg/buildtrees/libcap/arm-linux-dbg/libcap/../libcap/include/uapi/linux/capability.h
grep -E '^#define\s+CAP_([^\s]+)\s+[0-9]+\s*$' include/uapi/linux/capability.h | sed -e 's/^#define\s\+/{"/' -e 's/\s*$/},/' -e 's/\s\+/",/' -e 'y/ABCDEFGHIJKLMNOPQRSTUVWXYZ/abcdefghijklmnopqrstuvwxyz/' > cap_names.list.h
touch a.out | touch conftest | true -O2 -Dlinux -Wall -Wwrite-strings -Wpointer-arith -Wcast-qual -Wcast-align -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wshadow -Wunreachable-code  -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -I/home/igor/drawme/src/vcpkg/buildtrees/libcap/arm-linux-dbg/libcap/../libcap/include/uapi -I/home/igor/drawme/src/vcpkg/buildtrees/libcap/arm-linux-dbg/libcap/../libcap/include _makenames.c -o _makenames
./_makenames > cap_names.h
make[1]: Leaving directory '/home/igor/drawme/src/vcpkg/buildtrees/libcap/arm-linux-dbg/libcap'
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "your-project-name",
  "version-string": "0.1.0",
  "dependencies": [
    "boost-json",
    {
      "name": "google-cloud-cpp",
      "features": [
        "speech"
      ]
    },
    "curl",
    "sdl2",
    "sdl2-image",
    "sdl2-ttf"
  ]
}

```
</details>

--- a/ports/libcap/configure
+++ b/ports/libcap/configure
@@ -19,7 +19,7 @@ cat > Makefile.vcpkg <<END_MAKEFILE ;
 
 BUILD_OPTIONS = \
     "AR=$AR" \
-    "BUILD_CC=$CC_FOR_BUILD" \
+    "BUILD_CC=$CC" \
     "CC=$CC" \
     "RANLIB=$RANLIB" \
     "lib=lib" \

--- a/ports/libcap/vcpkg.json
+++ b/ports/libcap/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libcap",
   "version": "2.69",
-  "port-version": 2,
+  "port-version": 3,
   "description": "A library for getting and setting POSIX.1e (formerly POSIX 6) draft 15 capabilities.",
   "homepage": "https://git.kernel.org/pub/scm/libs/libcap/libcap.git",
   "license": "BSD-3-Clause OR GPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4126,7 +4126,7 @@
     },
     "libcap": {
       "baseline": "2.69",
-      "port-version": 2
+      "port-version": 3
     },
     "libcbor": {
       "baseline": "0.10.2",

--- a/versions/l-/libcap.json
+++ b/versions/l-/libcap.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "23bb9f2743e684bacba57b8cc8a931756f0e7447",
+      "version": "2.69",
+      "port-version": 3
+    },
+    {
       "git-tree": "d699cd38c2cf76ab59628fc3437c381ccabd2383",
       "version": "2.69",
       "port-version": 2

